### ISSUE TITLE
Svenska i koden

### ISSUE THREAD

**Issue by andersarpi** [Hanifb/marginal-calc#5]
Det spelar ingen roll att det är ett JS-lib utvecklat av svenskar, för svenskar, för att räkna ut information om svenska ekonomiska system. Koden och all text i kodfilerna skall ändå vara på engelska, inklusive kommentarer, variabel- och klassnamn och så vidare.

Det enda som är värre än svenska är blandat svenska och engelska, t.ex. params-kommentarerna i marginalCalc.js:

`* @param {Number} [priority = 100] Defines in what order the priority the module shall be calculated.`
`* @returns {marginalCalc} Den returnerar sig själv för att kunna kedja anrop.`

**Comment by Hanifb** [comment]
Skrev först alla docs i engelska, sen skrev jag om dem i svenska. Kan succeesivt skriva om docs:en till nästa version.

**Comment by perlun** [comment]
I agree, and I think we should probably also keep this discussion in English just to make it make more sense to foreigners who happen to read it as well. :smile:
